modules/extension/brewer/src/main/java/org/geotools/styling/builder/ContrastEnhancementBuilder.java

```diff
@@ -49,18 +53,38 @@ public ContrastEnhancementBuilder gamma(double gamma) {
         return gamma(literal(gamma));
     }
 
-    public ContrastEnhancementBuilder normalize() {
-        this.method = ContrastMethod.NORMALIZE;
-        this.unset = false;
-        return this;
+    public ContrastEnhancementBuilder normalize(Map<String, Expression> constrastParameters) {
+        return contrastMethod("normalize", constrastParameters);
```

Comment:
Nobody except CSS and Brewer modules should be using those methods, everyone else should 

    Normalize norm = new Normaliz();
    norm.setAlgorithm("StretchToMinimumMaximum");
    norm.addParameter("minValue",23.9);

etc